ui/src/translations/en.json

```diff
@@ -108,5 +108,8 @@
   "volume_statue_failed_pv_bound_hint": "The volume is bound to a storage request.",
   "delete_a_volume": "Delete a volume",
   "delete_a_volume_warning": "Deleting this volume will permanently delete the data it contains.",
-  "delete_a_volume_confirm": "Do you want to delete "
+  "delete_a_volume_confirm": "Do you want to delete ",
+  "select_a_storageClass": "Select a Storage Class...",
+  "select_a_type": "Select a type...",
```

Comment:
`a type` is vague, `a type of volume` would be better I think (ditto for the French version).

Reply:
You marked it as resolved but nothing changed.

Is there a reason to keep it as is (then you should have replied to the comment) or you simply forget a `git add`?

Reply:
oups. pushed :)

Reply:
:+1:

Reply:
It has been change in the lastest version :)